lib/voodoo_sms.rb

```diff
@@ -14,6 +14,14 @@ class RequiredParameter < StandardError; end
     class InvalidParameterFormat < StandardError; end
   end
 
+  class Response
+    attr_reader :reference_id
+
+    def initialize(reference_id)
+      @reference_id = reference_id
```

Comment:
@tomsabin I decided to go with reference_id so that the gem's API conforms with Voodoo's. What do you think?